check-vmware/vmware_logconf/local_config.ini

```diff
@@ -13,15 +13,15 @@ handlers=fileHandler
 
 [handler_stream_handler]
 class=StreamHandler
-level=DEBUG
+level=INFO
 formatter=formatter
 args=(sys.stderr,)
 
 [handler_fileHandler]
-class=FileHandler
-level=DEBUG
+class=logging.handlers.TimedRotatingFileHandler
+level=INFO
 formatter=formatter
-args=("vmware-checks-" + time.strftime("%Y-%m-%d") + ".log", "a")
+args=("vmware-checks.log", "D")
```

Comment:
Not adding date because now we are rotating logs?

Reply:
Yep exactly, logs will now be rotated daily, like the other logs in `/var/log/shinken`. I believe they will be cleaned automatically now too? But I'll have to did into logrotate more to see.